CMakeLists.txt

```diff
@@ -1,7 +1,19 @@
 cmake_minimum_required (VERSION 2.8.11)
+cmake_policy(SET CMP0048 NEW)
 
 # Defer enabling C and CXX languages.
-project (BoringSSL NONE)
+project (BoringSSL VERSION 1.0.0 LANGUAGES NONE)
```

Comment:
`project(... VERSION ...)` available since CMake 3.0 => `cmake_minimum_required(VERSION 3.0)` have to be used.

Reply:
Done.

Reply:
https://github.com/hunter-packages/boringssl/pull/1/commits/301c51321506c1f3e9c4ebbbfcc53f8d71227f02